tests/example_data_generator/management/commands/create-example-data.py

```diff
@@ -0,0 +1,165 @@
+
+LOREM_IPSUM = [
+    """\
+    <p>Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod
+    tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim
+    veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea
+    commodi consequat. Quis aute iure reprehenderit in voluptate velit esse
+    cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat
+    cupiditat non proident, sunt in culpa qui officia deserunt mollit anim
+    id est laborum.</p>
+    """,
+
+    """\
+    <p>Sed ut perspiciatis, unde omnis iste natus error sit voluptatem
+    accusantium doloremque laudantium, totam rem aperiam eaque ipsa, quae
+    ab illo inventore veritatis et quasi architecto beatae vitae dicta
+    sunt, explicabo. Nemo enim ipsam voluptatem, quia voluptas sit,
+    aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos,
+    qui ratione voluptatem sequi nesciunt, neque porro quisquam est, qui
+    dolorem ipsum, quia dolor sit amet consectetur adipisci[ng] velit, sed
+    quia non numquam [do] eius modi tempora inci[di]dunt, ut labore et
+    dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam,
+    quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut
+    aliquid ex ea commodi consequatur? Quis autem vel eum iure
+    reprehenderit, qui in ea voluptate velit esse, quam nihil molestiae
+    consequatur, vel illum, qui dolorem eum fugiat, quo voluptas nulla
+    pariatur?</p>
+    """,
+
+    """\
+    <p>At vero eos et accusamus et iusto odio dignissimos ducimus, qui
+    blanditiis praesentium voluptatum deleniti atque corrupti, quos dolores
+    et quas molestias excepturi sint, obcaecati cupiditate non provident,
+    similique sunt in culpa, qui officia deserunt mollitia animi, id est
+    laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita
+    distinctio. Nam libero tempore, cum soluta nobis est eligendi optio,
+    cumque nihil impedit, quo minus id, quod maxime placeat, facere
+    possimus, omnis voluptas assumenda est, omnis dolor repellendus.
+    Temporibus autem quibusdam et aut officiis debitis aut rerum
+    necessitatibus saepe eveniet, ut et voluptates repudiandae sint et
+    molestiae non recusandae. Itaque earum rerum hic tenetur a sapiente
+    delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut
+    perferendis doloribus asperiores repellat…</p>
+    """,
+]
+
+STAFF_USER_USERNAME = 'admin{}'
+DEFAULT_USER_USERNAME = 'user{}'
+PASSWORD = 'password'
+
+
+class Command(BaseCommand):
+    """
+    Command to create example data for OpenSlides.
+    """
+    help = 'Create example data for OpenSlides.'
+
+    chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-t', '--topics',
+            default=100,
+            type=int,
+            help='Number of topics to be created.',
```

Comment:
It would be helpful to get the default number in the help output. Also for other arguments.